cmd/client-s3.go

```diff
@@ -940,7 +940,11 @@ func isAmazonAccelerated(host string) bool {
 }
 
 func isGoogle(host string) bool {
-	return s3utils.IsGoogleEndpoint(url.URL{Host: host})
+	url, err := url.Parse(host)
```

Comment:
@CarterMcClellan thanks for this PR.

`isGoogle()` is used to receive only a host address without scheme and path. 

Now the problem is that, if we go with this change and then move to go 1.8, we will have the following (unexpected, I know) result:

`isGoogle("storage.googleapis.com") = false`

That's because with go 1.8, `url.Parse("storage.googleapis.com")` returns `url.Host = ""` and `url.Path = "storage.googleapis.com"`

So I really suggest moving url.Parse() code in config-host-add.go with the change that you already did.